brainiak/eventseg/event.py

```diff
@@ -60,14 +60,12 @@ class EventSegment(BaseEstimator):
     n_iter: int : default 500
         Maximum number of steps to run during fitting
 
+    event_chains: ndarray with length = n_events
+        Array with unique value for each separate chain of events, each linked
+        in the order they appear in the array
```

Comment:
As far as I can tell, this is the first time the term "chain" is used. I think it would help to provide a definition.

Reply:
Good idea, I added more info to the top of the file